content/blog/01_mapmaking.md

```diff
@@ -0,0 +1,44 @@
++++
+title = "Map Making"
+date = 2021-05-01
+
+extra.image = "assets/images/blog/mapmaking.png"
++++
+
+Map Making 101
+
+When a builder decides that they want to make a map, they first of all, head to our (whitelisted) build server found at <code>build.nucleoid.xyz</code>
+
+They choose a game and create a map using a handy command.
+
+<br>
+<img class="header_image" src="https://iili.io/Kg34kl.jpg">
```

Comment:
Images would rather be done through markdown if possible- also, can we move the images into the static content on the site so that it doesn't have to go off-site?